### DISCUSSION TITLE
Please Help

### DISCUSSION THREAD

**Discussion by cty7** [community/community#35961] [Codespaces]
![Screen Shot 2022-10-12 at 7 01 55 PM](https://user-images.githubusercontent.com/112792430/195463910-ceb850fd-d132-4856-b4a7-ff896fad4dca.png)

**Comment by Nishu0** [comment]
@cty7 
Try to contact support team.
Other solution might be that the problem was with 'xyz' website and that website might be down.
This error is from server side.
